test/mini-graphql-http-client.js

```diff
@@ -93,4 +104,93 @@ describe("MiniGraphqlHttpClient tests", () => {
             expect(cache2.map.get("2421565178")).to.have.deep.property("json", { bla: 1 });
         });
     });
+
+    describe("#retry", () => {
+        it("should try making 6 retries on 5xx responses but stop on 5-th try because of good response", async function () {
+            let wasError = false;
+            let numberOfRetries = 6;
+            const cache = {};
+            const fakeFetch = createFakeFetch([500, 500, 500, 500, 200, 500]);
+            const client = MiniGraphqlHttpClient({
+                uri: "https://a.aa",
+                fetch: fakeFetch,
+                cache,
+                retry: numberOfRetries,
+            });
+
+            await client.query({ query: "{ bla }" }).catch(() => {
+                wasError = true;
+            });
+
+            expect(fakeFetch.calls).to.be.equal(5);
+            expect(wasError).to.be.false;
+        });
+
+        it("should make 3 retries on 5xx error response and return error on 3 retry, it's will be 4-th try", async function () {
+            let wasError = false;
+            let numberOfRetries = 3;
+            const cache = {};
+            const fakeFetch = createFakeFetch([500, 500, 500, 500, 200]);
+            const client = MiniGraphqlHttpClient({
+                uri: "https://a.aa",
+                fetch: fakeFetch,
+                cache,
+                retry: numberOfRetries,
+            });
+
+            await client.query({ query: "{ bla }" }).catch(() => {
+                wasError = true;
+            });
+
+            expect(fakeFetch.calls).to.be.equal(4);
+            expect(wasError).to.be.true;
+        });
+
+        it("should not make retries on 4xx error response and return error after 1st try", async function () {
+            let errorText;
+            let wasError = false;
+            let numberOfRetries = 3;
+            const cache = {};
+            const fakeFetch = createFakeFetch([400, 500, 200]);
+            const client = MiniGraphqlHttpClient({
+                uri: "https://a.aa",
+                fetch: fakeFetch,
+                cache,
+                retry: numberOfRetries,
+            });
+
+            await client.query({ query: "{ bla }" }).catch((error) => {
+                errorText = error.message;
+                wasError = true;
+            });
+
+            expect(errorText).to.be.equal("400 Bad Request");
+            expect(wasError).to.be.true;
+            expect(fakeFetch.calls).to.be.equal(1);
+        });
+
+        it("should make 5 retries on fetch error and return good response on 5 fetch", async function () {
```

Comment:
I think instead of "make 5 retries" it should say "make 5 tries" ?
Also, looks like we have identical unit test above. Right?